CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # [vNext]
 
+## Breaking changes:
```

Comment:
Ok. So according to the versioning policy, I would mention this in the `## New Features` section, instead of the `## Breaking changes` and would merge into something like this:

```
* Reqnroll.Verify: Support for Verify v24 (Verify.Xunit v24.2.0) for .NET 4.7.2+ and .NET 6.0+. For earlier versions of Verify or for .NET 4.6.2, use the latest 2.0.3 version of the plugin that is compatible with Reqnroll v2.*. (#151)
```